.github/workflows/bootstrap.yml

```diff
@@ -0,0 +1,71 @@
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions
+
+name: Bootstrap
+on:
+  - pull_request # without merge conflicts
+  - push # branch or tag
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  main:
+    strategy:
+      fail-fast: false
+      matrix:
+        version: [
+                                                                2.076.1, 2.077.1, 2.078.3, 2.079.1,
+          2.080.1, 2.081.2, 2.082.1, 2.083.1, 2.084.1, 2.085.1, 2.086.1, 2.087.1, 2.088.1, 2.089.1,
+          2.090.1, 2.091.1, 2.092.1, 2.093.1, 2.094.2, 2.095.1, 2.096.1, 2.097.2, 2.098.1, 2.099.1,
+          2.100.2, 2.101.2, 2.102.2, 2.103.1, 2.104.2, 2.105.3, 2.106.1, 2.107.1
+        ]
+
+    name: Build with dmd-${{ matrix.version }}
+    runs-on: ubuntu-22.04
+    timeout-minutes: 40
```

Comment:
Looks like you can be waaay less conservative for the time-out. :)

Reply:
10 minutes it is then!